conf.yaml

```diff
@@ -126,7 +126,8 @@ contents:
             prefix:     en/welcome-to-elastic
             current:    *stackcurrent
             index:      welcome-to-elastic/index.asciidoc
-            branches:   [ 8.3 ]
+            branches:   [ 8.4 ]
+            live:       *stacklivemain
```

Comment:
`*stacklivemain` indicates that `main`, `8.4`, and `7.17` are live. But we're only publishing the 8.4 branch (`branches:   [ 8.4 ]`). My guess would be that live needs to be updated to something like `live:    [ 8.4 ]`.

Reply:
Thanks. I'll try that.

Reply:
Yep, what Brandon said!